src/routes/page/myProfile/+page.svelte

```diff
@@ -0,0 +1,87 @@
+<script>
+	import { PUBLIC_BRAND_NAME, PUBLIC_LOGIN_IMAGE } from '$env/static/public';
+	import { Row, Col, Card, CardBody, CardTitle, Table } from '@sveltestrap/sveltestrap';
+	import Breadcrumb from '$lib/common/Breadcrumb.svelte';
+	import HeadTitle from '$lib/common/HeadTitle.svelte';
+	import { onMount } from 'svelte';
+	import { myInfo } from '$lib/services/auth-service';
+    import Loader from '$lib/common/Loader.svelte';
+
+	/**
+	 * @type {{ full_name: any; id?: string; user_name?: string | undefined; first_name?: string | undefined; last_name?: string | undefined; email?: string | undefined; role?: string | undefined; avatar?: string | undefined; color?: string | undefined; token?: string | undefined; }}
```

Comment:
This syntax is not looking good. Can we define like this? 
/** @type {Object} CurrentUserModel
@property {string} [user_name] */